### PR TITLE
Legend and popups added back

### PR DESCRIPTION
**Pull request by DamonU2**

### CODE REVIEW

**Comment by DamonU2** [comment]
@jvanulde

File: docs/_pages/en/dsra_scenario_map.md [hunk 1]
```diff
@@ -153,6 +154,11 @@ crossorigin=""></script>
         sauidLayer.setFeatureStyle( selection, selectedStyle(), 100 );
       });
 
+      // Add a popup with desired property
+      L.popup().setContent( "<strong>Personnes déplacées après 90 jours: </strong>" + e.layer.properties.sCt_Res90_b0.toString() )
```
Comment (jvanulde) [inline docs/_pages/en/dsra_scenario_map.md:158]: Is this supposed to be in English?

**Review by jvanulde — CHANGES_REQUESTED** [review]
Just one potential change.

File: docs/_pages/en/dsra_scenario_map.md (hunk 1 above)
Comment (DamonU2) [reply docs/_pages/en/dsra_scenario_map.md]: Fixed. I am apparently very sloppy today. Thanks for catching that.

**Review by jvanulde — APPROVED** [review]
Thanks @DamonU2, looks good.

**Pull request merged**